frb_example/dart_minimal/lib/main.dart

```diff
@@ -1,8 +1,11 @@
 import 'package:frb_example_dart_minimal/src/rust/api/minimal.dart';
 import 'package:frb_example_dart_minimal/src/rust/frb_generated.dart';
 
+final LOGGER = FRBLogger.init_logger();
```

Comment:
Hmm, this does not seem to be super perfect API... I hope the `RustLib.init` is enough to be the only function called to initialize everything.

To allow initialization logic to be called, maybe we can add support to some attribute like `#[frb(init_dart="this_Dart_code_will_be_called_when_frb_init")]` and use that to init logging.

Reply:
:) That bothered me as well :)

That was the `ergonomics improvement` I did today. Now all can be configured via parameters to the macro. However, optionally a user can still call `FRBLogger.init_logger();` for customization - but doesn't have to.

When a user provides a custom log function he probably wants to write it in dart - thus he can use this function instead of providing a custom log function to the macro in rust.